BUILDING.md

```diff
@@ -290,9 +290,14 @@ $ CI_JS_SUITES=child-process CI_NATIVE_SUITES= make coverage
 The above command executes tests for the `child-process` subsystem and
 outputs the resulting coverage report.
 
-The `make coverage` command downloads some tools to the project root directory
-and overwrites the `lib/` directory. To clean up after generating the coverage
-reports:
+Alternatively, for the JavaScript test suite, it's faster to run:
+
+```text
+$ CI_JS_SUITES=fs-mkdir CI_NATIVE_SUITES= make jscov
```

Comment:
Nit: A note about unsetting `CI_NATIVE_SUITES` would help the readers.

Reply:
Perhaps:

_Note: `CI_NATIVE_SUITES` is unset, ensuring that the `fs-mkdir` tests are the only tests executed, otherwise the default suite of native tests will also be run._

Reply:
Yup, that makes it crystal clear 🙂

Reply:
I'm not sure we should recommend the use of `CI_*` variables to the public. AFAICT they are meant for internal use by our CI systems, and should be considered "private".

Reply:
@refack we reference this variable earlier in the document already:

```
CI_JS_SUITES=child-process CI_NATIVE_SUITES= make coverage
```

and it's a nice way to kick off a partial suite of tests with coverage; which is nice, because it's somewhat involved:

1. you need to set the `NODE_V8_COVERAGE` variable to the appropriate path.
2. you need to know to run `make js-coverage-report`.

Long story short I'd like to expose something like this behavior to users, open to alternate suggestions for implementation.